### ISSUE TITLE
Godot crashed when trying to past a node.

### ISSUE THREAD

**Issue by carllacan** [godotengine/godot#100194]
### Tested versions

v4.4.dev4.official [36e6207bb]

### System information

Godot v4.4.dev4 - Manjaro Linux #1 SMP PREEMPT_DYNAMIC Mon, 25 Nov 2024 05:29:44 +0000 on X11 - X11 display driver, Multi-window, 3 monitors - Vulkan (Forward+) - dedicated AMD Radeon RX 550 / 550 Series (RADV POLARIS12) - AMD Ryzen 5 3600 6-Core Processor (12 threads)

### Issue description

I was trying to copy and paste a node. I might have mitakenly had text in the copypaste clipboard, rather than the node I wanted to copy.

Godot crashed after pressing ctrl+v. Here's the resulting log:

```
================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.4.dev4.official (36e6207bb48fafd43b98a458d0c461ad982b4e68)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] /usr/lib/libc.so.6(+0x3d1d0) [0x7218a7bfd1d0] (??:0)
[2] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x4a5ca72] (??:0)
[3] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x48fbb7f] (??:0)
[4] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x2604592] (??:0)
[5] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x18c91ce] (??:0)
[6] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x18ee5ca] (??:0)
[7] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x405f014] (??:0)
[8] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x4264c7d] (??:0)
[9] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x283b437] (??:0)
[10] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x286232f] (??:0)
[11] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x289f437] (??:0)
[12] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x26604ed] (??:0)
[13] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x26401e2] (??:0)
[14] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x49fbdeb] (??:0)
[15] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x4b832d] (??:0)
[16] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x3ff48d3] (??:0)
[17] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x3ff676f] (??:0)
[18] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x4bf218] (??:0)
[19] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x41c2e8] (??:0)
[20] /usr/lib/libc.so.6(+0x25e08) [0x7218a7be5e08] (??:0)
[21] /usr/lib/libc.so.6(__libc_start_main+0x8c) [0x7218a7be5ecc] (??:0)
[22] /home/carles/Godot/Godot4/Godot_v4.4/Godot_v4.4-dev4_linux.x86_64() [0x43af1a] (??:0)
-- END OF BACKTRACE --
================================================================
```

### Steps to reproduce

I haven't been able to reproduce the exact sequence of clicks that led to the crash.

### Minimal reproduction project (MRP)

I haven't been able to reproduce the exact sequence of clicks that led to the crash.

**Comment by AThousandShips** [comment]
Please try a more recent version to confirm this hasn't already been solved, like [4.4.dev6](https://godotengine.org/article/dev-snapshot-godot-4-4-dev-6/)

**Comment by carllacan** [comment]
> Please try a more recent version to confirm this hasn't already been solved, like [4.4.dev6](https://godotengine.org/article/dev-snapshot-godot-4-4-dev-6/)

Try as I might I haven't been able to reproduce it, even in this version, so if nothing of interest can be seen in the backtrace then I guess this should be closed.

**Comment by AThousandShips** [comment]
Since the trace was made with a build without debug symbols it unfortunately doesn't provide any useful information 

If you can reproduce this again in the future please comment with more details here and this can be reopened
